### ISSUE TITLE
Plain text files/non app bundles that have been code signed do not retain the code sign extended attributes

### ISSUE THREAD

**Issue by carlashley** [packagesdev/packages#30]
When building a package that contains a code signed file that is not a mach-o binary, or app bundle, the extended attributes that contain the code sign details are not kept when the package is installed.

For example, a shell script `example.sh` that has been code signed and gets installed to `/usr/local/bin/example.sh` should retain the code signing details.

The extended attributes that are missing are (based on a `codesign` test run on macOS 10.14 GM, Xcode 10 installed):
```
com.apple.cs.CodeDirectory
com.apple.cs.CodeRequirements
com.apple.cs.CodeRequirements-1
com.apple.cs.CodeSignature
```

The script is code signed using (example only):
```
[carl@pegasus]:outset # codesign -s "Mac Developer: foo@example.org (ABC01FFFGH)" -i com.github.example example.sh
[carl@pegasus]:outset # codesign -dr - example.sh 
Executable=example.sh
designated => identifier "com.github.example" and anchor apple generic and certificate leaf[subject.CN] = "Mac Developer: foo@example.org (ABC01FFFGH)" and certificate 1[field.1.2.345.678901.234.5.6.7] /* exists */
[carl@pegasus]:outset # xattr example.sh 
com.apple.cs.CodeDirectory
com.apple.cs.CodeRequirements
com.apple.cs.CodeRequirements-1
com.apple.cs.CodeSignature
```
Being able to keep these extended attributes will be very useful for deploying scripts that are code signed.

**Comment by packagesdev** [comment]
Interesting question. I don't have an answer right now.

o With Packages, the only extended attributes that are currently preserved are com.apple.FinderInfo and com.apple.ResourceFork using an AppleDoubleFormat file. 

o I seem to remember some crashes of pax or ditto on some macOS releases when trying to archive files with extended attributes other than these 2.

**Comment by packagesdev** [comment]
This should be addressed by version 1.2.6 which is entering the final tests phase.

**Comment by packagesdev** [comment]
1.2.6  (547)
